client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -202,6 +204,33 @@ class PurchasesListing extends Component {
 		);
 	}
 
+	getProductActionButtons( purchase ) {
+		const { translate, selectedSiteSlug: site } = this.props;
+		const actionButton = this.getActionButton( purchase );
+
+		let serviceButton = null;
+		if ( isJetpackBackup( purchase ) ) {
+			serviceButton = (
+				<Button href={ `/activity-log/${ site }?group=rewind` } compact>
+					{ translate( 'View backups' ) }
+				</Button>
+			);
+		} else if ( isJetpackScan( purchase ) ) {
+			serviceButton = (
+				<Button href={ `/activity-log/${ site }?group=rewind` } compact>
+					{ translate( 'View threats' ) }
```

Comment:
Since we don't know if a site has threats we shouldn't tell the user to go see them. How about View scan results?

Reply:
@enejb - We don't show that, though? The only thing that shows in the activity log is a threat if one is present.